stepper-2.0.md

```diff
@@ -66,17 +66,16 @@ Protocol
 
 **Stepper step (relative move)**
 
-Position is specified as a 32-bit signed long.
+Steps to move is specified as a 32-bit signed long.
 
 The speed value is a float composed of a 23-bit significand (mantissa) and a 4-bit exponent
```

Comment:
For just one of the 3 sections that describes how the float is composed, can you provide an example for a specific number or two (maybe 500.0 and 0.000234 or something like that) how this would be broken out into the 3 parts specified in the table?